src/codecs2/field_codec_default.h

```diff
@@ -127,9 +127,14 @@ namespace dccl
                   FieldCodecBase::require(max() <= boost::numeric::bounds<WireType>::highest(),
                                           "(dccl.field).max must be <= maximum of this field type.");
 
+                  // allowable epsilon for min / max to diverge from nearest quantile
+                  const double min_max_eps = 1e-10;
+                  
+                  // ensure that max and min are multiples of the resolution chosen
```

Comment:
Would it make sense to require that (max-min) is an exact multiple of the resolution chosen, to allow configurations like {min: 0.5, max: 10.5, resolution: 1}?